src/js/utils/log.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-console */
+
+const noop = () => {
+    // noop
+};
```

Comment:
Why define `noop` upfront?

Inline avoided early assignment and scope polution.

Reply:
Good call, hoisting the func sails cap'n.